src/hotspot/cpu/x86/stubGenerator_x86_64_cbrt.cpp

```diff
@@ -49,142 +49,142 @@
 /* Represents 0x7FFFFFFFFFFFFFFF double precision in lower 64 bits*/
 ATTRIBUTE_ALIGNED(16) static const juint _ABS_MASK[] =
 {
-    4294967295, 2147483647, 0, 0
```

Comment:
The "L" suffix means different things for Windows (MSVC) and gcc/clang, and like the `long`
type, should rarely (perhaps never?) be used in HotSpot code.  `long` is a 32bit type on Windows
and 32bit gcc/clang platforms, and a 64bit type on 64bit gcc/clang platforms. (Or something like
that. I'm not sure what happens when building for windows with gcc/clang, which isn't a
supported configuration anyway.)

Reply:
I was surprised to discover that the "L" suffix is used quite a lot. I think those are likely often mistakes.

Reply:
So, we could have just used "U" suffix to guarantee 32-bit unsigned integer types because 32-bit platforms are no longer supported in OpenJDK?